101-vm-simple-rhel/azuredeploy.json

```diff
@@ -24,25 +24,31 @@
       "metadata": {
```

Comment:
You should probably inline the default value ("7.2") into the "storageProfile"'s "sku" property. Users probably won't change this often enough for it to be a parameter or a variable, and if it's inlined, then the user should be auto-completion from the JSON schema to help them know which "sku" values are supported.